runtime/compiler/p/env/J9CPU.cpp

```diff
@@ -93,7 +93,7 @@ CPU::hasPopulationCountInstruction()
 #if defined(J9OS_I5)
    return false;
 #else
-   return TR::Compiler->target.cpu.isAtLeast(TR_PPCp7);
+   return TR::Compiler->target.cpu.id() >= TR_PPCp7;
```

Comment:
I've just looked at a proposed PR that replaces `TR::Compiler->target.cpu.id() >= TR_PPCp7;` with `TR::Compiler->target.cpu.isAtLeast(TR_PPCp7);`
This PR seems to be doing the opposite

Reply:
isAtLeast() takes in an enum, we have a new api `isAtLeast()` that takes in the new processor enum type and an old api `isAtLeast()` that takes in the old processor enum type. Certain compilers with certain platform don't seem to distinguish between enums so they will treat the two as the same function and complain there is a function name clash. To solve this, I need to get rid of the old isAtLeast() api. And this was the one of the only places that still uses the old api. So this change is to remove the old api and make sure it doesn't break the build.

Once that PR is merged, this change will get overwritten by the Power changes.